Makefile

```diff
@@ -1,38 +1,40 @@
 # Make targets for the Exchange REST API server
 
 SHELL = /bin/bash -e
-DOCKER_REGISTRY ?= openhorizon
+
+# Some of these vars are also used by the Dockerfiles
 ARCH ?= amd64
+COMPILE_CLEAN ?= clean
 DOCKER_NAME ?= amd64_exchange-api
-VERSION = $(shell cat src/main/resources/version.txt)
-DOCKER_NETWORK=exchange-api-network
+DOCKER_NETWORK = exchange-api-network
+DOCKER_REGISTRY ?= openhorizon
+EXCHANGE_VERSION = $(shell cat src/main/resources/version.txt)
```

Comment:
@naphelps Can this be reverted back to `VERSION`? There are build pipelines that refer to that and it would also be consistent with Anax and CSS/ESS as well.